Dockerfile

```diff
@@ -1,20 +1,17 @@
 FROM mcr.microsoft.com/dotnet/core/sdk:2.2 AS build
 WORKDIR /app
 
-# copy csproj and restore as distinct layers
-COPY *.csproj .
-RUN dotnet restore
-
-# copy everything else and build app
+# copy and build app
 COPY . ./
```

Comment:
Ideally we shouldn't copy the tests project but I tried "COPY ntbs-service/ ./" and similarly for EFAuditer but that seemed to be wrong

Reply:
Should it just be copy ntbs-service/* actually to copy the whole folder and everything in it?